readme.md

```diff
@@ -1,5 +1,13 @@
 # Microsoft Azure IoT SDK for .NET
 
+## 2.0.0 clients release notice
+
+A [2.0.0 preview release](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2022-10-14-v2preview) has been published for each package in this library. 
+With this release, all future features will be brought only to this new major version, so users are highly encouraged to migrate from the 1.X.X releases to 
```

Comment:
Don't encourage users to migrate while it is in preview.

Instead, we want them to try it out and give us feedback.